go/pserver/client/c/test/test_train.py

```diff
@@ -1,5 +1,23 @@
 import paddle.v2 as paddle
 import paddle.v2.dataset.uci_housing as uci_housing
+import paddle.v2.master as master
+import os
+import cPickle as pickle
+
+etcd_ip = os.getenv("MASTER_IP", "127.0.0.1")
+etcd_endpoint = "http://" + etcd_ip + ":2379"
+
+
+def cloud_reader():
```

Comment:
We can put this into `paddle.reader.recordio` in the future, so `paddle.reader.recordio` can support both local and cloud reader. (not necessary in this PR).